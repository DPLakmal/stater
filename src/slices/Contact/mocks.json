[
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "default",
    "primary": {
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Get in Touch",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Have questions or need more information? Reach out to us.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "form_success_message": {
        "__TYPE__": "FieldContent",
        "value": "die",
        "type": "Text"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Eiusmod aute",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "map_iframe_url": {
        "__TYPE__": "LinkContent",
        "value": {
          "__TYPE__": "ExternalLink",
          "url": "https://slicemachine.dev"
        }
      },
      "recipient_email_address": {
        "__TYPE__": "FieldContent",
        "value": "shot",
        "type": "Text"
      }
    },
    "items": []
  }
]
