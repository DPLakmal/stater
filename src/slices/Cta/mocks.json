[
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "default",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "cta_title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "heading2",
            "content": {
              "text": "Church"
            }
          }
        ]
      },
      "cta_description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Reprehenderit commodo eu et mollit voluptate exercitation nisi proident."
            }
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Right",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Primary"
                }
              ],
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Twitter"
                }
              ],
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "http://twitter.com",
                    "target": "_blank"
                  }
                }
              ]
            ]
          }
        ]
      }
    },
    "items": []
  },
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "fullWidth",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "value": "Full",
        "type": "Select"
      },
      "cta_title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "heading2",
            "content": {
              "text": "Speech 2",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "cta_description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Dolor velit ullamco quis quis exercitation incididunt proident qui."
            }
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "value": "band",
                  "type": "Text"
                }
              ],
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "http://google.com",
                    "target": "_blank"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Right",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Primary"
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Another"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Secondary"
                }
              ],
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "https://google.com",
                    "target": "_blank"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Left"
                }
              ]
            ]
          }
        ]
      }
    },
    "items": [
      {
        "__TYPE__": "GroupItemContent",
        "value": []
      }
    ]
  }
]
