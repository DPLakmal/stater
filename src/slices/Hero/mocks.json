[
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "variation2",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "value": "Full",
        "type": "Select"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Dorem ipsum",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Lorem ipsum dolor sit amet",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Korem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit interdum, ac aliquet odio mattis.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "http://google.com"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Right",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Primary",
                  "type": "Select"
                }
              ],
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Secondary"
                }
              ]
            ]
          }
        ]
      },
      "hero_image": {
        "origin": {
          "id": "main",
          "url": "https://images.unsplash.com/photo-1494173853739-c21f58b16055",
          "width": 3277,
          "height": 4092
        },
        "url": "https://images.unsplash.com/photo-1494173853739-c21f58b16055?rect=0%2C1316%2C3277%2C1460&w=1064&h=474",
        "width": 1064,
        "height": 474,
        "edit": {
          "zoom": 1,
          "crop": {
            "x": 0,
            "y": 1316
          },
          "background": "transparent"
        },
        "credits": null,
        "alt": null,
        "__TYPE__": "ImageContent",
        "thumbnails": {
          "mobile": {
            "origin": {
              "id": "mobile",
              "url": "https://images.unsplash.com/photo-1600861194802-a2b11076bc51",
              "width": 2545,
              "height": 2545
            },
            "url": "https://images.unsplash.com/photo-1600861194802-a2b11076bc51?rect=0%2C334%2C2545%2C1876&w=335&h=247",
            "width": 335,
            "height": 247,
            "edit": {
              "zoom": 1,
              "crop": {
                "x": 0,
                "y": 334
              },
              "background": "transparent"
            },
            "credits": null,
            "alt": null
          },
          "tablet": {
            "origin": {
              "id": "tablet",
              "url": "https://images.unsplash.com/photo-1498050108023-c5249f4df085",
              "width": 3882,
              "height": 2584
            },
            "url": "https://images.unsplash.com/photo-1498050108023-c5249f4df085?rect=0%2C353%2C3882%2C1879&w=688&h=333",
            "width": 688,
            "height": 333,
            "edit": {
              "zoom": 1,
              "crop": {
                "x": 0,
                "y": 353
              },
              "background": "transparent"
            },
            "credits": null,
            "alt": null
          }
        }
      }
    },
    "items": [
      {
        "__TYPE__": "GroupItemContent",
        "value": []
      }
    ]
  },
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "variation4",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "value": "None",
        "type": "Select"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "value": "Full",
        "type": "Select"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Dorem ipsum",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Lorem ipsum dolor sit amet.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Korem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit interdum, ac aliquet odio mattis.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "http://google.com"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Left",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Primary"
                }
              ],
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ]
            ]
          }
        ]
      },
      "hero_images": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "image",
                {
                  "origin": {
                    "id": "main",
                    "url": "https://images.unsplash.com/photo-1544731612-de7f96afe55f",
                    "width": 6000,
                    "height": 4000
                  },
                  "url": "https://images.unsplash.com/photo-1544731612-de7f96afe55f",
                  "width": 6000,
                  "height": 4000,
                  "edit": {
                    "zoom": 1,
                    "crop": {
                      "x": 0,
                      "y": 0
                    },
                    "background": "transparent"
                  },
                  "credits": null,
                  "alt": null,
                  "__TYPE__": "ImageContent",
                  "thumbnails": {}
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "image",
                {
                  "provider": "imgix",
                  "url": "https://images.unsplash.com/photo-1597589022928-bb4002c099ec?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHwzfHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzQwNjU5MjN8MA&ixlib=rb-4.0.3&q=85",
                  "alt": "yellow lemon fruit beside green leaves",
                  "origin": {
                    "id": "3t8rQ3LD2JI",
                    "url": "https://images.unsplash.com/photo-1597589022928-bb4002c099ec?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHwzfHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzQwNjU5MjN8MA&ixlib=rb-4.0.3&q=85",
                    "width": 3024,
                    "height": 3024
                  },
                  "width": 3024,
                  "height": 3024,
                  "edit": {
                    "background": "transparent",
                    "zoom": 1,
                    "crop": {
                      "x": 0,
                      "y": 0
                    }
                  },
                  "credits": null,
                  "__TYPE__": "ImageContent",
                  "thumbnails": {}
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "image",
                {
                  "provider": "imgix",
                  "url": "https://images.unsplash.com/photo-1603824231744-517ef271df7e?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHwyfHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzQwNjU5MjN8MA&ixlib=rb-4.0.3&q=85",
                  "alt": "sliced lemon fruit on blue surface",
                  "origin": {
                    "id": "SDX4_TQcxGU",
                    "url": "https://images.unsplash.com/photo-1603824231744-517ef271df7e?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHwyfHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzQwNjU5MjN8MA&ixlib=rb-4.0.3&q=85",
                    "width": 3690,
                    "height": 2470
                  },
                  "width": 3690,
                  "height": 2470,
                  "edit": {
                    "background": "transparent",
                    "zoom": 1,
                    "crop": {
                      "x": 0,
                      "y": 0
                    }
                  },
                  "credits": null,
                  "__TYPE__": "ImageContent",
                  "thumbnails": {}
                }
              ]
            ]
          }
        ]
      }
    },
    "items": [
      {
        "__TYPE__": "GroupItemContent",
        "value": []
      }
    ]
  },
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "variation3",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Dorem ipsum",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Porem ipsum dolor sit amet dolor.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "http://google.com"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Left",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Primary",
                  "type": "Select"
                }
              ],
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Secondary"
                }
              ]
            ]
          }
        ]
      },
      "hero_image": {
        "origin": {
          "id": "main",
          "url": "https://images.unsplash.com/photo-1551739440-5dd934d3a94a",
          "width": 3456,
          "height": 4320
        },
        "url": "https://images.unsplash.com/photo-1551739440-5dd934d3a94a?rect=0%2C0%2C3456%2C4320&w=1280&h=1600",
        "width": 1280,
        "height": 1600,
        "edit": {
          "zoom": 1,
          "crop": {
            "x": 0,
            "y": 0
          },
          "background": "transparent"
        },
        "credits": null,
        "alt": null,
        "__TYPE__": "ImageContent",
        "thumbnails": {}
      }
    },
    "items": [
      {
        "__TYPE__": "GroupItemContent",
        "value": []
      }
    ]
  },
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "default",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "type": "Select",
        "value": "Full"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "type": "Select",
        "value": "Full"
      },
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Lorem ipsum dolor sit amet.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Korem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit interdum, ac aliquet odio mattis.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Dorem ipsum",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ],
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "https://prismic.io"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Right",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Primary",
                  "type": "Select"
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "link",
                {
                  "__TYPE__": "LinkContent",
                  "value": {
                    "__TYPE__": "ExternalLink",
                    "url": "https://google.com",
                    "target": "_blank"
                  }
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Left"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Secondary",
                  "type": "Select"
                }
              ],
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Button Text"
                }
              ]
            ]
          }
        ]
      },
      "hero_image": {
        "origin": {
          "id": "main",
          "url": "https://images.unsplash.com/photo-1601933973783-43cf8a7d4c5f",
          "width": 6597,
          "height": 4398
        },
        "url": "https://images.unsplash.com/photo-1601933973783-43cf8a7d4c5f?rect=0%2C1%2C6597%2C4396&w=1280&h=853",
        "width": 1280,
        "height": 853,
        "edit": {
          "zoom": 1,
          "crop": {
            "x": 0,
            "y": 1
          },
          "background": "transparent"
        },
        "credits": null,
        "alt": null,
        "__TYPE__": "ImageContent",
        "thumbnails": {}
      }
    },
    "items": [
      {
        "__TYPE__": "GroupItemContent",
        "value": []
      }
    ]
  }
]
