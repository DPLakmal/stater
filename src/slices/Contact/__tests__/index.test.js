import "@testing-library/jest-dom";
import simplifyMockData from "@/utils/prismic/simplifyMockData";
import mocks from "../mocks.json";
import Component from "../index";
import { render } from "@testing-library/react";

describe("Contact Section", () => {
  // top padding
  it("first variation with full top padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.top_padding = "Full";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation with half top padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.top_padding = "Half";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation without top padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.top_padding = "None";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  // bottom padding
  it("first variation with full bottom padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.bottom_padding = "Full";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation with half bottom padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.bottom_padding = "Half";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation without bottom padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.bottom_padding = "None";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
});
