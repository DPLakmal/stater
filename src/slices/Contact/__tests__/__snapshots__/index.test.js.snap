// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Section first variation with full bottom padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="bg-highlight pt-10 md:pt-[60px] lg:pt-[80px]"
    >
      <div
        class="px-[16px] md:px-[40px] xl:px-[80px] pt-0 pb-[40px] md:pb-[80px] xl:pb-[120px] "
      >
        <div
          class="mx-auto w-full max-w-[1280px] "
        >
          <div
            class="grid grid-cols-1 gap-10 lg:grid-cols-2"
          >
            <div>
              <div
                class="mb-3"
              />
              <div
                class="mb-10"
              >
                <p
                  class=" mb-7 last:mb-0"
                />
              </div>
            </div>
            <div>
              <div
                class="border border-[#D5D8E7] max-w-[512px] mx-auto bg-white py-10 px-8 rounded-[20px] mt-8 md:mt-10 lg:mt-0 "
              >
                <h4
                  class="text-h4 text-center mb-2"
                >
                  Contact us
                </h4>
                <form>
                  <div />
                  <div />
                  <div />
                  <div />
                  <div
                    class="mt-4 text-center"
                  >
                    <a
                      class="mx-auto btn btn-primary"
                      href=""
                    >
                      Submit
                    </a>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full h-[250px] md:h-[350px] lg:h-[500px] relative mt-5 md:mt-10 lg:mt-[80px]"
      >
        <iframe
          class="h-full w-full absolute left-0 top-0"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          src="[object Object]"
          title="undefined-map-section"
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Contact Section first variation with full top padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="bg-highlight pt-10 md:pt-[60px] lg:pt-[80px]"
    >
      <div
        class="px-[16px] md:px-[40px] xl:px-[80px] pb-0 pt-[40px] md:pt-[80px] xl:pt-[120px] "
      >
        <div
          class="mx-auto w-full max-w-[1280px] "
        >
          <div
            class="grid grid-cols-1 gap-10 lg:grid-cols-2"
          >
            <div>
              <div
                class="mb-3"
              />
              <div
                class="mb-10"
              >
                <p
                  class=" mb-7 last:mb-0"
                />
              </div>
            </div>
            <div>
              <div
                class="border border-[#D5D8E7] max-w-[512px] mx-auto bg-white py-10 px-8 rounded-[20px] mt-8 md:mt-10 lg:mt-0 "
              >
                <h4
                  class="text-h4 text-center mb-2"
                >
                  Contact us
                </h4>
                <form>
                  <div />
                  <div />
                  <div />
                  <div />
                  <div
                    class="mt-4 text-center"
                  >
                    <a
                      class="mx-auto btn btn-primary"
                      href=""
                    >
                      Submit
                    </a>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full h-[250px] md:h-[350px] lg:h-[500px] relative mt-5 md:mt-10 lg:mt-[80px]"
      >
        <iframe
          class="h-full w-full absolute left-0 top-0"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          src="[object Object]"
          title="undefined-map-section"
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Contact Section first variation with half bottom padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="bg-highlight pt-10 md:pt-[60px] lg:pt-[80px]"
    >
      <div
        class="px-[16px] md:px-[40px] xl:px-[80px] pt-0 pb-[20px] md:pb-[40px] xl:pb-[60px] "
      >
        <div
          class="mx-auto w-full max-w-[1280px] "
        >
          <div
            class="grid grid-cols-1 gap-10 lg:grid-cols-2"
          >
            <div>
              <div
                class="mb-3"
              />
              <div
                class="mb-10"
              >
                <p
                  class=" mb-7 last:mb-0"
                />
              </div>
            </div>
            <div>
              <div
                class="border border-[#D5D8E7] max-w-[512px] mx-auto bg-white py-10 px-8 rounded-[20px] mt-8 md:mt-10 lg:mt-0 "
              >
                <h4
                  class="text-h4 text-center mb-2"
                >
                  Contact us
                </h4>
                <form>
                  <div />
                  <div />
                  <div />
                  <div />
                  <div
                    class="mt-4 text-center"
                  >
                    <a
                      class="mx-auto btn btn-primary"
                      href=""
                    >
                      Submit
                    </a>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full h-[250px] md:h-[350px] lg:h-[500px] relative mt-5 md:mt-10 lg:mt-[80px]"
      >
        <iframe
          class="h-full w-full absolute left-0 top-0"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          src="[object Object]"
          title="undefined-map-section"
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Contact Section first variation with half top padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="bg-highlight pt-10 md:pt-[60px] lg:pt-[80px]"
    >
      <div
        class="px-[16px] md:px-[40px] xl:px-[80px] pb-0 pt-[20px] md:pt-[40px] xl:pt-[60px] "
      >
        <div
          class="mx-auto w-full max-w-[1280px] "
        >
          <div
            class="grid grid-cols-1 gap-10 lg:grid-cols-2"
          >
            <div>
              <div
                class="mb-3"
              />
              <div
                class="mb-10"
              >
                <p
                  class=" mb-7 last:mb-0"
                />
              </div>
            </div>
            <div>
              <div
                class="border border-[#D5D8E7] max-w-[512px] mx-auto bg-white py-10 px-8 rounded-[20px] mt-8 md:mt-10 lg:mt-0 "
              >
                <h4
                  class="text-h4 text-center mb-2"
                >
                  Contact us
                </h4>
                <form>
                  <div />
                  <div />
                  <div />
                  <div />
                  <div
                    class="mt-4 text-center"
                  >
                    <a
                      class="mx-auto btn btn-primary"
                      href=""
                    >
                      Submit
                    </a>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full h-[250px] md:h-[350px] lg:h-[500px] relative mt-5 md:mt-10 lg:mt-[80px]"
      >
        <iframe
          class="h-full w-full absolute left-0 top-0"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          src="[object Object]"
          title="undefined-map-section"
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Contact Section first variation without bottom padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="bg-highlight pt-10 md:pt-[60px] lg:pt-[80px]"
    >
      <div
        class="px-[16px] md:px-[40px] xl:px-[80px] pt-0 pb-0 "
      >
        <div
          class="mx-auto w-full max-w-[1280px] "
        >
          <div
            class="grid grid-cols-1 gap-10 lg:grid-cols-2"
          >
            <div>
              <div
                class="mb-3"
              />
              <div
                class="mb-10"
              >
                <p
                  class=" mb-7 last:mb-0"
                />
              </div>
            </div>
            <div>
              <div
                class="border border-[#D5D8E7] max-w-[512px] mx-auto bg-white py-10 px-8 rounded-[20px] mt-8 md:mt-10 lg:mt-0 "
              >
                <h4
                  class="text-h4 text-center mb-2"
                >
                  Contact us
                </h4>
                <form>
                  <div />
                  <div />
                  <div />
                  <div />
                  <div
                    class="mt-4 text-center"
                  >
                    <a
                      class="mx-auto btn btn-primary"
                      href=""
                    >
                      Submit
                    </a>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full h-[250px] md:h-[350px] lg:h-[500px] relative mt-5 md:mt-10 lg:mt-[80px]"
      >
        <iframe
          class="h-full w-full absolute left-0 top-0"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          src="[object Object]"
          title="undefined-map-section"
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Contact Section first variation without top padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="bg-highlight pt-10 md:pt-[60px] lg:pt-[80px]"
    >
      <div
        class="px-[16px] md:px-[40px] xl:px-[80px] pt-0 pb-0 "
      >
        <div
          class="mx-auto w-full max-w-[1280px] "
        >
          <div
            class="grid grid-cols-1 gap-10 lg:grid-cols-2"
          >
            <div>
              <div
                class="mb-3"
              />
              <div
                class="mb-10"
              >
                <p
                  class=" mb-7 last:mb-0"
                />
              </div>
            </div>
            <div>
              <div
                class="border border-[#D5D8E7] max-w-[512px] mx-auto bg-white py-10 px-8 rounded-[20px] mt-8 md:mt-10 lg:mt-0 "
              >
                <h4
                  class="text-h4 text-center mb-2"
                >
                  Contact us
                </h4>
                <form>
                  <div />
                  <div />
                  <div />
                  <div />
                  <div
                    class="mt-4 text-center"
                  >
                    <a
                      class="mx-auto btn btn-primary"
                      href=""
                    >
                      Submit
                    </a>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full h-[250px] md:h-[350px] lg:h-[500px] relative mt-5 md:mt-10 lg:mt-[80px]"
      >
        <iframe
          class="h-full w-full absolute left-0 top-0"
          loading="lazy"
          referrerpolicy="no-referrer-when-downgrade"
          src="[object Object]"
          title="undefined-map-section"
        />
      </div>
    </div>
  </section>
</div>
`;
